prosper/common/prosper_logging.py

```diff
@@ -418,6 +402,7 @@ class DiscordWebhook(object):
 
     """
     __base_url = 'https://discordapp.com/api/webhooks/'
+    __webhook_url_format = __base_url + r"(\d+)/(\w+)"
```

Comment:
That should probably be `([\w-]+)` then?, since `\w.+` would match a single 'word' character followed by 1 or more 'any' character. Or could just make the whole thing: `r"(\d+)/(.+)"`. to match any string after the second `/`.